modules/tools/00_mod.mk

```diff
@@ -170,14 +171,20 @@ tools += yamlfmt=v0.17.2
 tools += kubeconform=v0.7.0
 
 # https://pkg.go.dev/k8s.io/code-generator/cmd?tab=versions
-K8S_CODEGEN_VERSION := v0.33.3
```

Comment:
Have you looked at the potential consequences of deleting this variable? It's a breaking change.

Reply:
Oh right my bad.. totally spaced out on that.. although if we keep that variable how do we update the env through renovate?

Reply:
We probably need to modify the regex. 🤣 But if the variable isn't in use across the org, we can remove it. But that needs to be checked.

Reply:
yes I can check that.. I really dont want to touch the working regex 😛